scrapers/victrola.py

```diff
@@ -0,0 +1,80 @@
+from bs4 import BeautifulSoup
+import requests
+from models import Coffee
+import re
+import logging
+
+# scraping Victrola roasters
+
+
+def scrape_victrola():
+    roaster = 'Victrola'
+    victrola = 'http://www.victrolacoffee.com/collections/all-coffee-offerings'
+    r = requests.get(victrola)
+    soup = BeautifulSoup(r.content)
+
+    coffees_for_sale = soup.find_all('a', {'class':'product-link'})
+    total_coffees = len(coffees_for_sale)
+    coffees_entered = 0
+    error_coffees = []
+    for item in coffees_for_sale:
+        name,description,notes,region,status,size, product_url = [""]*7
+        price = int()
+        url = item['href']
+        product_url = 'http://www.victrolacoffee.com' + url
+        logging.info("Getting url: {}".format(url))
+        r = requests.get(product_url)
+        coffee_soup = BeautifulSoup(r.content)
+        name = coffee_soup.h2.string
+        if 'Subscription' in name:
+            total_coffees-=1
+            continue
+        try:
+            price = float(coffee_soup.find(itemprop='price').string.strip()[2:])
+            status = 'Available'
+        except:
+            status = "Sold Out"
+        d = coffee_soup.find(itemprop='description').find_all('span')
+        if 'Blend' in name:
+            # different stuff for blends
+            notes = ''
+            region = ''
+            for x in d:
+                description += x.string
+        else:
+            # sometimes tasting notes just alone
+            try:
+                notes = coffee_soup(text=re.compile('Flavor:'))[1].string.strip()[8:]
+            except:
+                notes = coffee_soup.find(text="Tasting Notes").next_element.strip()[2:]
+                pass
+            try:
+                region = coffee_soup(text=re.compile(r'Region:'))[1][8:]
+            except:
+                region = 'n/a'
+                pass
+        try:
+            size = coffee_soup.find('select').option.string[:4]
```

Comment:
I mean for example. http://www.victrolacoffee.com/collections/all-coffee-offerings/products/victrola-roasters-choice-coffee-subscription should not be counted as a coffee since it has no size